.travis/package_steady_cli.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+echo "Packaging CLI"
+
+if [ -z $1 ];
+then
+  echo "You did not specify a release (format: X.Y.Z)"
+  exit 1
+fi
+
+cd `dirname $0`
+VULAS_RELEASE="$1"
+mkdir -p /tmp/cli
+WORKDIR="/tmp/cli"
+
+mkdir -p  $WORKDIR/steady-cli/app
+mkdir -p  $WORKDIR/steady-cli/instr
+
+mv ../lang-java/target/lang-java-${VULAS_RELEASE}-jar-with-dependencies.jar $WORKDIR/steady-cli/instr
+mv ../cli-scanner/target/cli-scanner-${VULAS_RELEASE}-jar-with-dependencies.jar $WORKDIR/steady-cli/steady-cli-${VULAS_RELEASE}-jar-with-dependencies.jar
+cp vulas-custom.properties $WORKDIR/steady-cli/vulas-custom.properties
```

Comment:
Sorry for bugging yet another time, but the `.sample` postfix was actually quite nice... Could you please rename and I will then approve the PR

Reply:
Done